src/js/core/widget/core/Marquee.js

```diff
@@ -234,7 +234,7 @@
 						value,
 						returnValue;
 
-					value = state * (textWidth - containerWidth);
+					value = state * (textWidth - containerWidth + 25);
```

Comment:
I've applied your PR and rebuild TAU. With your change I can see last character of text on right side (no cutting) in UI Components app. 
Please have in mind that this change is made to core (src/js/core/widget/core/Marquee.js) implementation and will applied for both Mobile and Wearable profiles. Such hard coded values may bring unexpected result in some cases. Can we find alternative way to fix it? For example by applying percentage etc.

Reply:
I measured the right gradient size at 15%.
and the size was divided by two and divided evenly.
I tested it on the Marquee Sample, the Snap Marquee List, and the Marquee List, and they all confirmed normal operation.